web/src/navbar_alerts.ts

```diff
@@ -147,8 +151,31 @@ export function get_demo_organization_deadline_days_remaining(): number {
     return days_remaining;
 }
 
+function is_browser_timezone_same_as(uncanonicalized_target_timezone: string): boolean {
+    if (uncanonicalized_target_timezone === "") {
+        return false;
+    }
+    const canonocalized_browser_timezone = new Intl.DateTimeFormat("en-US", {
+        timeZone: timerender.display_time_zone,
+    }).resolvedOptions().timeZone;
+    const canonicalized_target_timezone = new Intl.DateTimeFormat("en-US", {
+        timeZone: uncanonicalized_target_timezone,
+    }).resolvedOptions().timeZone;
+    return canonocalized_browser_timezone === canonicalized_target_timezone;
+}
+
+export function should_offer_to_update_timezone(): boolean {
+    // This offer is only for logged-in users with the setting enabled.
+    return Boolean(
```

Comment:
This is already a boolean and does not need to be converted.

```suggestion
    return (
```

Reply:
Somehow eslint wasn't happy about it, so I had to put a `Boolean` there

![image](https://github.com/user-attachments/assets/dff3b079-eda0-4bae-8b21-18e8fc463474)

Reply:
Do you see this error from `tools/lint -g frontend`? If not, your ESLint extension is probably stuck in an inconsistent state. Try **ESLint: Restart ESLint Server** in the command palette.

Reply:
I don't see it there.
I'll remove the `Boolean`